client/src/modules/needsDiagnosis/index.js

```diff
@@ -0,0 +1,105 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import {
+  NEEDS_DIAGNOSIS_REQUEST,
+  NEEDS_DIAGNOSIS_SUCCESS,
+  NEEDS_DIAGNOSIS_FAILURE,
+} from "../../constants/ActionTypes";
+import { CALL_API, GET } from "../../constants/Api";
+import { CHART_LINE } from "../../constants/Charts";
+import { ObjectNested, isEmptyObject, normalizDate } from "../../libraries";
+/* name of reducer */
+export const STATE_KEY = "needsdiagnosis";
+
+const initialState = {
+  stats: {},
+};
+
+/* reducer function */
+export default function reducer(state = initialState, action) {
```

Comment:
this would be a syntax error in python... i guess JS is ok with non-default params after a default param?

Reply:
sorry, but I don't understand, what do you mean after `non-default params`?

Reply:
By `non-default params`, I mean arguments without a default value. `(foo)` vs `(foo=bar)`. This was mostly a learning moment for me, nothing for you to change.